plugins/main/public/controllers/management/components/management/configuration/util-components/configuration-setting.js

```diff
@@ -10,10 +10,12 @@
  * Find more information about this on the LICENSE file.
  */
 
-import React, { Component, Fragment } from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-
+import { EuiAccordion, EuiBasicTable } from '@elastic/eui';
 import { EuiFieldText, EuiSpacer, EuiTextAlign } from '@elastic/eui';
```

Comment:
**Thought**: these imports could be in a single line, as they comes from the same place